client/wfnews-war/src/main/angular/tsconfig.json

```diff
@@ -6,7 +6,10 @@
     "typeRoots": [
       "node_modules/@types"
     ],
-    "baseUrl": "./",
+    "baseUrl": "./src",
```

Comment:
You shouldn't have to change the baseUrl, but unless you get any build weirdness it shouldn't matter